lib/project_types/script/layers/application/build_script.rb

```diff
@@ -5,16 +5,17 @@ module Layers
     module Application
       class BuildScript
         class << self
-          def call(ctx:, task_runner:, script_name:, extension_point_type:)
+          def call(ctx:, task_runner:, script_name:, description:, extension_point_type:)
             CLI::UI::Frame.open(ctx.message('script.application.building')) do
               begin
                 UI::StrictSpinner.spin(ctx.message('script.application.building_script')) do |spinner|
                   Infrastructure::PushPackageRepository.new(ctx: ctx).create_push_package(
-                    extension_point_type,
-                    script_name,
-                    task_runner.build,
-                    task_runner.compiled_type,
-                    task_runner.metadata
+                    extension_point_type: extension_point_type,
+                    script_name: script_name,
+                    description: description,
+                    script_content: task_runner.build,
+                    compiled_type: task_runner.compiled_type,
+                    metadata: task_runner.metadata
```

Comment:
This probably hints that we could start thinking about wrapping this in some sort of value object. What do you think? I don't think we need to do it in this PR but _eventually_, there are a couple of other places that might benefit from a value object too.

Reply:
I agree. The first three entries: `extension_point_type`, `script_name` and `description` are all available on the `ScriptProject` object. I think we could also just reference that object throughout the code and avoid passing its fields in everywhere